src/platform/libretro/libretro.c

```diff
@@ -102,6 +102,130 @@ static unsigned imcapWidth;
 static unsigned imcapHeight;
 static size_t camStride;
 static bool envVarsUpdated;
+static unsigned frameskipType;
+static unsigned frameskipThreshold;
+static uint16_t frameskipCounter;
+static bool retroAudioBuffActive;
+static unsigned retroAudioBuffOccupancy;
+static bool retroAudioBuffUnderrun;
+static unsigned retroAudioLatency;
+static bool updateAudioLatency;
+
+/* Maximum number of consecutive frames that
+ * can be skipped */
+#define RETRO_FRAMESKIP_MAX 30
+
+/* Frame skipping functions */
+
+static void _retroAudioBuffStatusCallback(
+		bool active, unsigned occupancy, bool underrun_likely) {
+	retroAudioBuffActive    = active;
+	retroAudioBuffOccupancy = occupancy;
+	retroAudioBuffUnderrun  = underrun_likely;
+}
+
+static void _initFrameskip(void) {
```

Comment:
It'd be nice if you used the same style as the rest of the code.

Reply:
I do apologise for the oversight...

Unfortunately your coding style is quite discriminatory. I have vision problems, and your brace formatting and camel case variables are difficult for me to read and follow. I was already stressed about making my code compliant (I made several mistakes as a direct result of this, which took me a while to debug...), and I guess this variable slipped under the radar.

I have updated the code - I hope it meets your satisfaction.